src/python/espressomd/shapes.py

```diff
@@ -41,6 +41,7 @@ class Cylinder(ScriptInterfaceHelper):
 
     """
     _so_name = "Shapes::Cylinder"
+    _so_bind_methods = ['calc_distance']
```

Comment:
Couldn't you just introduce a base class an put the meme there instead of repeating it for every shape?

Reply:
Done.